containers/Dockerfile

```diff
@@ -37,10 +37,10 @@ ENV PROJECT_DIR=/projects \
 # install TA-Lib and other prerequisites
 #
 
-RUN mkdir ${PROJECT_DIR} \
-    && apt-get -y update \
-    && apt-get -y install libfreetype6-dev libpng-dev libopenblas-dev liblapack-dev gfortran \
-    && curl -L http://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-src.tar.gz | tar xvz
+RUN mkdir ${PROJECT_DIR}
+RUN apt-get -y update
+RUN apt-get -y install libxft-dev libfreetype6-dev libpng-dev libopenblas-dev liblapack-dev gfortran
```

Comment:
I cache these steps for debugging and frequent runs. If this was an `official` image I would  squash these to aid in others referencing this parent.